pages/common/updog.md

```diff
@@ -0,0 +1,25 @@
+# updog
+
+> A Replacement for Python's SimpleHTTPServer.
+> It allows uploading and downloading via HTTP/S, can set ad hoc SSL certificates and use HTTP basic auth.
+> More information: <https://github.com/sc0tfree/updog>.
+
+- Start a HTTP server on your current directory:
+
+`updog`
+
+- Start a HTTP server on a specified directory:
+
+`updog --directory {{/path/to/directory}}`
+
+- Start a HTTP server on a specified port:
+
+`updog --port {{port}}`
+
+- Start a HTTP server with a password [To login, you should leave the username blank and just enter the password in the password field]:
```

Comment:
Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
- Start a HTTP server with a password (To login, leave the username blank and enter the password in the password field):
```